logic/round.js

```diff
@@ -233,5 +253,37 @@ Round.prototype.land = function () {
 		}.bind(this));
 };
 
+/**
+ * Calls:
+ * - updateVotes
+ * - updateMissedBlocks
+ * - flushRound
+ * - applyRound
+ * - updateVotes
+ * - flushRound
+ * - restoreRoundSnapshot
+ * - restoreVotesSnapshot
+ * @implements {updateVotes}
+ * @implements {updateMissedBlocks}
+ * @implements {flushRound}
+ * @implements {applyRound}
+ * @implements {restoreRoundSnapshot}
+ * @implements {restoreVotesSnapshot}
+ * @return {function} call result
+ */
+Round.prototype.landBackward = function () {
```

Comment:
I would prefer to call: `Round.prototype.backwardLand` to equate with the existing `backwardTick`.